src/plots/mapbox/mapbox.js

```diff
@@ -747,17 +752,32 @@ proto.getView = function() {
     var mapCenter = map.getCenter();
     var center = { lon: mapCenter.lng, lat: mapCenter.lat };
 
+    var canvas = map.getCanvas();
+    var w = canvas.width;
+    var h = canvas.height;
+    var cUL = map.unproject([0, 0]).toArray();
+    var cUR = map.unproject([w, 0]).toArray();
+    var cLR = map.unproject([w, h]).toArray();
+    var cLL = map.unproject([0, h]).toArray();
+    var coordinates = [cUL, cUR, cLR, cLL];
+
     return {
         center: center,
         zoom: map.getZoom(),
         bearing: map.getBearing(),
-        pitch: map.getPitch()
+        pitch: map.getPitch(),
+        _derived: {
+            coordinates: coordinates
+        }
     };
 };
 
-proto.getViewEdits = function(cont) {
+proto.getViewEdits = function(cont, derived) {
     var id = this.id;
     var keys = ['center', 'zoom', 'bearing', 'pitch'];
+    if(derived === true) {
+        keys.push('_derived');
+    }
```

Comment:
I'm not a fan of adding boolean arguments to functions. 

By making

```js
getViewEdits({}, true)
```

you don't know what the `true` mean until you scroll down to the `getViewEdits` definition.

I would prefer turning `getViewEdits({}, true)` into `getViewEdits({}, {derived: true})` or split `getViewEdits` into a version w/o the `_derived` container and one (e.g. named `getViewEditsWithDerived`) with it.

Reply:
I went with `getViewEditsWithDerived` in 45ddd806fd4679486c71dd52ebc92c6af5079688.